cloud/features.rst

```diff
@@ -16,6 +16,20 @@ High Availability
 
 In addition to continuous protection which is explained above, high availability is available if your application requires less exposure to downtime. If at provisioning you select high availability we provision stand-bys. This can be for your primary node, or for your distributed nodes.
 
+Security
```

Comment:
There is a pretty high ratio of headers to content in this section. Maybe you could remove the Encryption and TFA headers and leave just the paragraphs.